scrapy/exporters.py

```diff
@@ -195,12 +207,12 @@ def export_item(self, item):
         fields = self._get_serialized_fields(item, default_value='',
                                              include_empty=True)
         values = list(self._build_row(x for _, x in fields))
-        self.csv_writer.writerow(values)
+        self._writerow(values)
 
     def _build_row(self, values):
         for s in values:
             try:
-                yield to_native_str(s)
+                yield to_bytes(s) if six.PY2 else s
```

Comment:
hm, why is this needed?
unless I'm missing something, `to_native_str()` is equivalent.